Lib/crypt.py

```diff
@@ -30,7 +30,13 @@ def mksalt(method=None, *, log_rounds=12):
     if not method.ident:
         s = ''
     elif method.ident[0] == '2':
```

Comment:
I don't understand this test. Which methods are supposed to be matched? Would you mind to add just one example in a comment? Same question for "elif method.ident in ('5', '6')" below.